docs/BR.md

```diff
@@ -1142,10 +1142,11 @@ The validity interval of an OCSP response is the difference in time between the
 
 For the status of Subscriber Certificates:
 
-1. OCSP responses MUST have a validity interval greater than or equal to eight hours;
-2. OCSP responses MUST have a validity interval less than seven days;
-3. For OCSP responses whose validity interval is less than or equal to sixteen hours, the CA SHALL update the information provided via an Online Certificate Status Protocol at least eight hours prior to the nextUpdate;
-4. For OCSP responses whose validity interval is greater than sixteen hours, the CA SHALL update the information provided via an Online Certificate Status Protocol prior to one-half of the validity period remaining before the nextUpdate.
+1.  OCSP responses for certificates must meet the following requirements:
+	a. Minimum validity of eight (8) hours; maximum validity of ten (10) days for leaf certificates and 12 months for subordinate CA certificates; and
+	b. If the validity period of the OCSP response is less than 16 hours, then the next response must be available at ½ the validity period.  If the validity period is greater than 16 hours then the next response must be available no less than 8 hours before the current response expires.
```

Comment:
From your e-mail, it sounds like Microsoft is in the process of updating its policy to be more relaxed. Structurally, I think the formatting I proposed is a bit clearer.

I'll open up a PR on your branch (I'm getting GitHub errors at the moment), but I think the language is

```suggestion
1. OCSP responses MUST have a validity interval of greater than or equal to eight hours;
2. OCSP responses MUST have a validity internal of less than or equal to ten days;
3. For OCSP responses with validity intervals less than sixteen hours, then the CA SHALL update the information provided via an Online Certificate Status Protocol prior to one-half of the validity period before the nextUpdate.
4. For OCSP responses with validity intervals greater than or equal to sixteen hours, then the CA SHALL update the information provided via an Online Certificate Status Protocol at least eight hours prior to the nextUpdate.
```

That is, it keeps the substance of your changes, but clarifies what "current response expires" means by using the language from the OCSP RFC.

The subordinate CA certificate requirement is captured on line 1151

Reply:
Also, I realize Microsoft is in the process of updating its Root Policy, but one nice thing about the existing (7 day max, 1/2 update) is that it more naturally aligned with the CRL requirements within Section 4.9.7. of the BRs

That is, the existing (part of the BRs) requirement states:
> If the CA publishes a CRL, then the CA SHALL update and reissue CRLs at least once every seven days, and the value of the nextUpdate field MUST NOT be more than ten days beyond the value of the thisUpdate field.

That is, phrased differently:
1. CRLs have a maximum validity internal of 10 days
2. New CRLs MUST be published at least every seven days

For OCSP, the existing BRs have:
1. OCSP responses have a maximum validity of 10 days
2. New OCSP responses must be published every 4 days

For OCSP, the existing Microsoft Root Policy has
1. OCSP responses have a maximum validity of 7 days
2. New OCSP responses must be published no later than every 3.5 days

The proposed new Microsoft policy would be.
1. OCSP responses have a maximum validity of 10 days
2. New OCSP responses must be published no later than every 9.66 days (**BUG?** to be more lax than BRs)